src/OrchardCore.Modules/OrchardCore.OpenId/Startup.cs

```diff
@@ -66,10 +72,29 @@ public override void ConfigureServices(IServiceCollection services)
                 ServiceDescriptor.Scoped<IPermissionProvider, Permissions>(),
                 ServiceDescriptor.Scoped<INavigationProvider, AdminMenu>(),
             });
+
+            services.Configure<CookiePolicyOptions>(options =>
+            {
```

Comment:
you will probably tell me it doesn't work @MatthijsKrempel but was it not possible to just add

```
services.ConfigureApplicationCookie(options => options.Cookie.SameSite = SameSiteMode.None)
```

here to alter the settings all ready configured by users, rather than having to use the cookie middleware to write it?

Reply:
@deanmarcussen sounds good, doesnt work.

Reply:
What I do need to do though, is move all the backwards compatibility into this module.
As we only need it here.